source/wp-content/themes/wporg-developer/inc/user-content.php

```diff
@@ -176,6 +176,8 @@ public static function scripts_and_styles() {
 				array(
 					'copy'   => __( 'Copy', 'wporg' ),
 					'copied' => __( 'Code copied', 'wporg' ),
+					'expand'   => __( 'Expand full source code', 'wporg' ),
+					'collapse' => __( 'Collapse source code', 'wporg' ),
```

Comment:
Can we make the text just `Expand/Collapse` or `Expand All /Collapse All`, it seems to be a big button to me for with the current text. Or there're other reasons we using these texts?

Reply:
Sure, I just copied the existing text. I'll make that change before merging this.